CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 #   idf.py build
 
 set(min_supported_idf_version "5.3.0")
```

Comment:
Should it be 5.5.0?
```suggestion
set(min_supported_idf_version "5.5.0")
```

Reply:
5.3 is now breaking due to uart: https://github.com/espressif/arduino-esp32/actions/runs/16365909770/job/46242981911?pr=10884#step:5:1988

Please fix :)